### PR TITLE
Tidy up serializer comparison table in README.md

### PR DESCRIPTION
**Pull request by PawelLipski**
Spurred by today's comments by Hydra team.

### CODE REVIEW

**Comment by MarconZet** [comment]
I think the phrase "standard for scala code" doesn't do justice to how runtime safe it is, the prime example being codecs in Borer.

**Comment by PawelLipski** [comment]
> I think the phrase "standard for scala code" doesn't do justice to how runtime safe it is, the prime example being codecs in Borer.

Ok, reverted to _encoders and decoders are checked during compile time_ for Borer and Circe

File: README.md [hunk 1]
```diff
@@ -82,13 +87,14 @@ It may also be worth including additional codecs for common types that are missi
...
+| Runtime safety         | none, uses reflection                                                                                                                                                                                                                                                                                                                                                                           | encoders and decoders are checked during compile time                          | standard for Scala code                                                                        | standard for Scala code                                                                          | encoders and decoders are checked during compile time                                                                             | depends on whether codecs were derived in compile time (then standard for Scala code), or not (than none)                                                    |
```
Comment (MarconZet) [inline README.md:98]: Actually, I don't think it's possible to create Avro or Protubuff definition that won't serialise, so four serialisers are runtime safe

Comment (PawelLipski) [reply README.md:98]: Okay... how would you suggest to rephrase it for Avro/Protobuf and/or Borer/Circe? 🤔

Comment (MarconZet) [reply README.md:98]: Maybe:
* encoders and decoders are created during compilation
* *.proto files are validated before compilation
* Avro schema is created during compilation

Comment (PawelLipski) [reply README.md]: Done, patched

**Review by MarconZet — APPROVED** [review]

**Pull request merged**